feature-libs/product-configurator/common/shared/testing/common-configurator-test-utils.service.ts

```diff
@@ -62,4 +62,27 @@ export class CommonConfiguratorTestUtilsService {
         htmlElement.innerHTML
     );
   }
+
+  /**
+   * Helper function for proving how many times the element comes in the DOM tree.
+   *
+   * @param {any} expect - Expectation for a spec.
+   * @param {Element} htmlElement - HTML element.
+   * @param {string} querySelector - Query selector
+   * @param {number} expectedNumber- expected number of elements
+   */
+  static expectNumberOfElements(
```

Comment:
This method is very helpful, for the new conflict and error message component we could also check the number of messages via this method.

Reply:
Glad to be helpful :)